mediatek: add Comfast CF-E395AX support by adding an alternative model name

db260179 opened the pull request:
both these devices share the board and same config, just different model number

Install instructions are the same as the CF-E393AX commit - https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=d8f4453bf2de9fd9baf3d660ed12e0797ff2cfdb

Neustradamus commented:
@db260179: Good job!

hauke commented on target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi:
```diff
@@ -0,0 +1,249 @@
+
+		port@1 {
+			reg = <1>;
+			label = "eth1";
```
Why did you add this `port@1` configuration?

db260179 replied:
So my other PR  - https://github.com/openwrt/openwrt/pull/16138 had identified that we where missing the config for the wan port.

This commit just aligns with that change

hauke replied:
Please do not hide such changes. When you do more than just moving the code add a extra commit

db260179 replied:
No intention of hiding anything?
Please advise what i should do? change this line and then do a second commit again?

db260179 replied:
@hauke As per your request, i have removed the seperate changes and will submit a seperate patch for those changes once this change is approved.

hauke commented on target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts:
```diff
@@ -1,247 +1,9 @@
-	nvmem-cell-names = "eeprom";
-};
-
+};
```
Please add a new line at the end of the file and also the other files.

db260179 replied:
Yep, will do

hauke commented:
What is the difference between the two devices?

db260179 commented:
> What is the difference between the two devices?

Just the model number, reason for this common config - the chassis is literally the only difference - they cost reduced it, so the 393ax has a bigger case than the 395ax - to board is identical.

hauke commented:
Do we really need an extra image and device tree?
If it is just about the name you can add an alternative name:
```
--- a/target/linux/mediatek/image/filogic.mk
+++ b/target/linux/mediatek/image/filogic.mk
@@ -482,6 +482,8 @@ TARGET_DEVICES += cmcc_rax3000m
 define Device/comfast_cf-e393ax
   DEVICE_VENDOR := COMFAST
   DEVICE_MODEL := CF-E393AX
+  DEVICE_ALT0_VENDOR := COMFAST
+  DEVICE_ALT0_MODEL := CF-E395AX
   DEVICE_DTS := mt7981a-comfast-cf-e393ax
   DEVICE_DTS_DIR := ../dts
   DEVICE_DTC_FLAGS := --pad 4096
```

db260179 commented:
> Do we really need an extra image and device tree? If it is just about the name you can add an alternative name:
> 
> ```
> --- a/target/linux/mediatek/image/filogic.mk
> +++ b/target/linux/mediatek/image/filogic.mk
> @@ -482,6 +482,8 @@ TARGET_DEVICES += cmcc_rax3000m
>  define Device/comfast_cf-e393ax
>    DEVICE_VENDOR := COMFAST
>    DEVICE_MODEL := CF-E393AX
> +  DEVICE_ALT0_VENDOR := COMFAST
> +  DEVICE_ALT0_MODEL := CF-E395AX
>    DEVICE_DTS := mt7981a-comfast-cf-e393ax
>    DEVICE_DTS_DIR := ../dts
>    DEVICE_DTC_FLAGS := --pad 4096
> ```

Yes makes sense, only issue the users might not know that this image will work on the e395ax, but i have changed the commit to your recommended change.

db260179 committed:
mediatek: add Comfast CF-E395AX support by adding an alternative model name

both these devices share the board and same config, just different model number

Install instructions are the same as the CF-E393AX commit - https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=d8f4453bf2de9fd9baf3d660ed12e0797ff2cfdb

Signed-off-by: David Bentham <db260179@gmail.com>

blogic commented:
merged, Thanks !

Neustradamus commented:
@blogic: Are you sure?

I do not see here:
- https://github.com/openwrt/openwrt/blob/main/target/linux/mediatek/image/filogic.mk

Neustradamus commented:
Merged commit:
- https://github.com/openwrt/openwrt/commit/794291bbdf26ad2be7581fc0c921e4d820937c79
